packages/core/useWebNotification/index.ts

```diff
@@ -0,0 +1,120 @@
+import { onMounted, Ref, ref, onUnmounted } from 'vue-demi'
+
+interface WebNotificationOptions {
+  /*
+   *
+   * The body string of the notification as specified in the constructor's options parameter.
```

Comment:
Can you follow the TSDocs format from other composables and remove the extract spaces?

Reply:
Will do. Will push shortly.